code/go/0chain.net/chaincore/chain/protocol_view_change.go

```diff
@@ -152,15 +152,15 @@ func (mc *Chain) isRegisteredEx(ctx context.Context, getStatePath func(n *node.N
 			return false
 		}
 
-		if list == nil {
-			return false
-		}
-
-		if err = allNodesList.Decode(list.Encode()); err != nil {
-			logging.Logger.Error("failed to decode block state node",
-				zap.Any("error", err))
-			return false
-		}
+		//if list == nil {
```

Comment:
> It might be an idea to cearafully merge in recent staging changes. I saw one possable issue with [storagesc/stakepool.go](https://github.com/0chain/0chain/pull/1033/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#diff-ce2ed4fb820ddf8bc8517075d5242e73b2e7547262aababb0be2e39ea481a56dR28) there could be others.

Yes, basically could happen if I did some changes here, and then this code was removed from staging. Don't know why the git didn't detect the conflicts. Anyway, it should be safe as this code is unused, I'll fix them.